dev/repo/seeds.exs

```diff
@@ -0,0 +1,91 @@
+defmodule Demo.Seeder do
+  require Ash.Query
+  alias Ash.Seed
+  alias Demo.Accounts.User
+  alias Demo.Tickets.{Customer, Organization, Representative, Ticket}
+
+  @spec insert_admin!(String.t(), String.t()) :: User.t()
+  def insert_admin!(first_name, last_name) do
+    Seed.seed!(User, %{
+      first_name: first_name,
+      last_name: last_name,
+      admin: true
+    })
+  end
+
+  @spec insert_user!(String.t(), String.t(), String.t(), String.t()) :: User.t()
+  def insert_user!(first_name, last_name, bio, history) do
+    Seed.seed!(User, %{
+      first_name: first_name,
+      last_name: last_name,
+      profile: %{
+        bio: bio,
+        history: history,
+      },
+      alternate_profiles: []
+    })
+  end
+
+  @spec insert_customer!(String.t(), String.t()) :: Customer.t()
+  def insert_customer!(first_name, last_name) do
+    Seed.seed!(Customer, %{
+      first_name: first_name,
+      last_name: last_name,
+      representative: false
+    })
+  end
+
+   @spec insert_representative!(String.t(), String.t()) :: Representative.t()
+  def insert_representative!(first_name, last_name) do
+    Seed.seed!(Representative, %{
+      first_name: first_name,
+      last_name: last_name,
+      representative: true
+    })
+  end
+
+  @spec insert_organization!(String.t()) :: Organization.t()
+  def insert_organization!(name) do
+    Seed.seed!(Organization, %{name: name});
+  end
+
+  @spec insert_ticket!(String.t(), String.t(), Customer.t(), Representative.t(), Organization.t()) :: Ticket.t()
+  def insert_ticket!(subject, description, reporter, representative, organization) do
+    Seed.seed!(Ticket, %{
+      subject: subject,
+      description: description,
+      reporter_id: reporter.id,
+      representative_id: representative.id,
+      organization_id: organization.id,
+      comments: []
+    })
+  end
+
+  def arguments("--truncate") do
```

Comment:
Is this hooked up? I don't see what calls it. If we were to use Ash actions we could actually support `upsert?: true` which could potentially prevent the need to truncate at all. We could also add `Ash.Seed.upsert`